syft/pkg/cataloger/java/archive_parser.go

```diff
@@ -16,7 +22,7 @@ import (
 	"github.com/anchore/syft/syft/pkg/cataloger/generic"
 )
 
-var _ generic.Parser = parseJavaArchive
+const MavenBaseURL = "https://repo1.maven.org/maven2"
```

Comment:
Should this also be configurable? E.g. if the user has a private maven repo, they could try to point at that to resolve licenses, which also could work for air-gap

Reply:
updated this to configurable with the above value as the default